.github/workflows/ci.yaml

```diff
@@ -26,14 +26,14 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v4
-            - uses: actions/setup-go@v5
-              with:
-                go-version: '>=1.22'
             - uses: bazel-contrib/setup-bazel@0.8.5
               with:
                 bazelisk-cache: true
                 repository-cache: true 
-            - run: go install github.com/hay-kot/scaffold@${{ matrix.scaffold-version }}
+            - uses: jaxxstorm/action-install-gh-release@25d5e2dd555cd74f1fab9ac1e6ea117acde2c0c4 # v1.12.0
```

Comment:
Have you used this anywhere else yet? Do we want to test it before putting it into a template and spreading it around?

Reply:
ah, this file is not part of the template. It's outside the `{{.ProjectSnake}}` folder so it's private only to us